### PR TITLE
Feature/fixes

### PR DESCRIPTION
**Pull request by FinalAngel**
adds nice icons/preview for all text enabled plugins

image

### CODE REVIEW

File: aldryn_bootstrap3/cms_plugins.py [hunk 1]
```diff
@@ -71,6 +70,8 @@ def render(self, context, instance, placeholder):
         context.update({'instance': instance})
         return context
 
+    def icon_src(self, instance):
```
Comment (intellectronica) [inline aldryn_bootstrap3/cms_plugins.py]: you are passing `instance` but not using it. is that to satisfy a pre-defined interface?

Comment (FinalAngel) [inline aldryn_bootstrap3/cms_plugins.py]: icon_src requires the instance argument

Comment (intellectronica) [inline aldryn_bootstrap3/cms_plugins.py]: k

**Comment by intellectronica** [comment]
:+1:

**Pull request merged**